### PR TITLE
Various fixed for placing new buffers into same arena

### PR DESCRIPTION
**Pull request by LLITCHEV**

### CODE REVIEW

File: tile/targets/cpu/jit.cc [hunk 1]
```diff
@@ -1683,6 +1721,8 @@ llvm::JITSymbol Runtime::findSymbol(const std::string& name) {
       {"_libxsmm_smmdispatch", symInfo(libxsmm_smmdispatch)},
       {"libxsmm_dmmdispatch", symInfo(libxsmm_dmmdispatch)},
       {"_libxsmm_dmmdispatch", symInfo(libxsmm_dmmdispatch)},
+      {"RunTimeLogEntry", symInfo(rt::RunTimeLogEntry)},   // For debugging
+      {"_RunTimeLogEntry", symInfo(rt::RunTimeLogEntry)},  // For debugging
```
Comment (mars1026) [inline tile/targets/cpu/jit.cc]: Is the second entry really necessary? There is logic a few lines down which is intended to handle the platform-dependent leading "_" character.

File: tile/targets/cpu/jit.cc [hunk 2]
```diff
@@ -216,7 +219,6 @@ Compiler::Compiler(llvm::LLVMContext* context, const Config& config)
 }
 
 ProgramModule Compiler::CompileProgram(const stripe::Block& program) {
-  IVLOG(4, program);
```
Comment (mars1026) [inline tile/targets/cpu/jit.cc]: seems harmless, easy to restore later if useful

File: tile/targets/cpu/jit.cc [hunk 3]
```diff
@@ -202,10 +204,11 @@ class Compiler : private stripe::ConstStmtVisitor {
   std::map<std::string, Scalar> scalars_;
   std::map<std::string, Buffer> buffers_;
   std::map<std::string, Index> indexes_;
+  uint64_t arenaSize_;
```
Comment (mars1026) [inline tile/targets/cpu/jit.cc]: given that there are two different constructor methods, perhaps it'd be safer to use value-initialization here?

File: tile/targets/cpu/jit.cc (hunk 1 above)
Comment (LLITCHEV) [reply tile/targets/cpu/jit.cc]: It was necessary. I tried without it and the linker couldn't find the method to link with. So, I added the second entry.

File: tile/targets/cpu/jit.cc (hunk 2 above)
Comment (LLITCHEV) [reply tile/targets/cpu/jit.cc]: Added it back in.

File: tile/targets/cpu/jit.cc (hunk 3 above)
Comment (LLITCHEV) [reply tile/targets/cpu/jit.cc]: Good catch! Thanks!

**Pull request merged**